libvips/iofuncs/type.c

```diff
@@ -217,6 +217,14 @@ vips_area_unref(VipsArea *area)
 		g_mutex_unlock(area->lock);
 }
 
+/* autoptr needs typed versions of functions for free.
+ */
+void
+VipsArrayDouble_unref(VipsArrayDouble *array)
+{
+    vips_area_unref(VIPS_AREA(array));
```

Comment:
```suggestion
	vips_area_unref(VIPS_AREA(array));
```
This would (hopefully) appease clang-format.